impl_core/src/main/java/io/opencensus/implcore/stats/MutableViewData.java

```diff
@@ -206,7 +214,11 @@ private CumulativeMutableViewData(View view, Timestamp start) {
     }
 
     @Override
-    void record(TagContext context, double value, Timestamp timestamp) {
+    void record(
+        TagContext context,
+        double value,
+        Timestamp timestamp,
+        @javax.annotation.Nullable Map<String, String> attachments) {
```

Comment:
Should there be a TODO here about storing the attachments?

Reply:
Done.